### DISCUSSION TITLE
pandoc.exe does not have embedded version information

### DISCUSSION THREAD

**Discussion by jonboede** [jgm/pandoc#9819] [General]
When inspecting the FileVersionInfo in pandoc.exe all the properties are blank.

It would be nice to know the version number of the installed version of Pandoc outside of Add/Remove Programs so that software using Pandoc can check to see if the installed version might need to be updated.

![Screenshot 2024-05-29 204644](https://github.com/jgm/pandoc/assets/171207837/8c5b9939-0c3b-4b7c-bd76-60cf4f324f33)

**Comment by jgm** [comment]
I'm not sure why the information doesn't appear there, but I'm pretty sure that the installer is aware of the installed version.   We do pass this information into the WiX program used to make the installer. (You could test this by trying to install an older version on top of a newer one, and see if it complains? I'm sorry, I don't have Windows and can't directly test.)

If anyone knows how we'd get this property to be present, let me know.  For the build procedure, see `.github/workflows/release-candidate.yml` and the files in `windows/`.
